README.md

```diff
@@ -51,78 +43,350 @@ A processing environment for HyP3 Plugins in AWS.
-- For Earthdata Cloud deployments:
-  - SSL certificate in AWS Certificate Manager for custom CloudFront domain name
-  - ID of the CloudFront Origin Access Identity used to access data in S3 
-- For non-Earthdata Cloud deployments:
-  - DNS record for custom API domain name
-  - SSL certificate in AWS Certificate Manager for custom API domain name
+We currently have HyP3 deployments in various AWS accounts managed by three different organizations,
+also referred to as "security environments" throughout our code and docs
+(because the AWS accounts have different security requirements depending on the organization):
+- ASF
+- EDC (Earthdata Cloud)
+- JPL
+- JPL-public 
 
-### Stack Parameters
-Review the parameters in [cloudformation.yml](apps/main-cf.yml) for deploy time configuration options.
+> [!IMPORTANT]
+> JPL deployments _must_ start with the JPL security environment, but can be migrated to `JPL-public`
+> after they are fully deployed and approved to have a public bucket.
 
-### Deploy with CloudFormation
+EDC deployment steps are not fully documented here.
+ASF and JPL deployment steps should be complete, so let us know if something is missing!
 
-To deploy HyP3 with reasonable defaults, follow the steps below. For more advanced
-deployment configuration, see the [deployment GitHub Action](.github/actions/deploy-hyp3/action.yml).
+For JPL, these deployment docs assume that:
+- the JPL account was set up in the "default" manner by the JPL cloud team
+- the developer deploying the account is able to log in with the `power_user` role
 
-From the repository root, 
+For a new EDC deployment, you need the following items (not necessarily a comprehensive list):
+- SSL certificate in AWS Certificate Manager for custom CloudFront domain name
+- ID of the CloudFront Origin Access Identity used to access data in S3
 
-- Install dependencies for build and run
-```sh
-make install
-```
+> [!TIP]
+> You can expand and collapse details specific to a security environment as you go through this README.
+> Make sure you're looking at the details for the security environment you're deploying into!  
 
-- Install Python dependencies for AWS Lambda functions (requires pip for python 3.13)
-```sh
-make build
-```
+### Set up 
+
+#### Ensure there is a CloudFormation templates bucket
+
+In order to deploy HyP3, the AWS account will need an S3 bucket to store AWS CloudFormation
+templates in the same region as the deployment. 
+
+For JPL and EDC security environments, this will likely have already been set up for you by their 
+respective cloud services team. You can confirm this by going to the AWS S3 console and looking 
+for a bucket named something like `cf-templates-<HASH>-<region>`. If not, follow the ASF steps below.
+
+<details>
+<summary>ASF: Create a CloudFormation templates bucket</summary>
+<br />
+
+*Note: This section only needs to be completed once per AWS account.*
```

Comment:
```suggestion
*Note: This section only needs to be completed once per region used in an AWS account.*
```
I think there's generally one per region, no? We just happen do mostly deploy to a single region.